src/services/cronJobs/checkProjectVerificationStatus.ts

```diff
@@ -173,6 +169,11 @@ const remindUpdatesOrRevokeVerification = async (project: Project) => {
   const user = await User.findOne({ id: Number(project.admin) });
 
   await sendProperNotification(project, project.verificationStatus as string);
+  await Promise.all([
```

Comment:
I think we must to it in `checkProjectVerificationStatus` in when  the list of projects to unverify are not empty.